economy/admin.py

```diff
@@ -1,3 +1,32 @@
 from django.contrib import admin
 
-# Register your models here.
+from economy.models import Transaction, Deposit, SociBankAccount, SociProduct
+
+admin.site.register(Transaction)
+admin.site.register(Deposit)
+
+
+class SociBankAccountAdmin(admin.ModelAdmin):
+    list_display = ['user', 'card_uuid', 'format_balance', 'sufficient_funds']
+
+    def format_balance(self, obj):
```

Comment:
What is this? It causes my admin to crash when adding a SociBankAccount.

Reply:
Old code I forgot to remove. Will fix.